<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QToolButton" name="toolButton_move">
        <property name="toolTip">
         <string>Move</string>
        </property>
        <property name="text">
         <string>Move</string>
        </property>
        <property name="icon">
         <iconset resource="cursor/cursor.qrc">
          <normaloff>:/move.svg</normaloff>:/move.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_create">
        <property name="toolTip">
         <string>Create</string>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
        <property name="icon">
         <iconset resource="cursor/cursor.qrc">
          <normaloff>:/create.svg</normaloff>:/create.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_link">
        <property name="toolTip">
         <string>Link</string>
        </property>
        <property name="text">
         <string>Link</string>
        </property>
        <property name="icon">
         <iconset resource="cursor/cursor.qrc">
          <normaloff>:/link.svg</normaloff>:/link.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_final">
        <property name="toolTip">
         <string>Halt</string>
        </property>
        <property name="text">
         <string>Toggle Final</string>
        </property>
        <property name="icon">
         <iconset resource="cursor/cursor.qrc">
          <normaloff>:/halt.svg</normaloff>:/halt.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_start">
        <property name="toolTip">
         <string>Start</string>
        </property>
        <property name="text">
         <string>Set Start</string>
        </property>
        <property name="icon">
         <iconset resource="cursor/cursor.qrc">
          <normaloff>:/first.svg</normaloff>:/first.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_rename">
        <property name="toolTip">
         <string>Rename</string>
        </property>
        <property name="text">
         <string>Rename</string>
        </property>
        <property name="icon">
         <iconset resource="cursor/cursor.qrc">
          <normaloff>:/rename.svg</normaloff>:/rename.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="DragWidget" name="widget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>548</width>
         <height>351</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>568</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
    <addaction name="actionSave_SVG"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave_SVG">
   <property name="text">
    <string>Save SVG</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DragWidget</class>
   <extends>QWidget</extends>
   <header>dragwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="cursor/cursor.qrc"/>
 </resources>
 <connections/>
</ui>
